RecoParticleFlow/PFProducer/src/PFEGammaAlgo.cc

```diff
@@ -2250,17 +2257,59 @@ buildRefinedSuperCluster(const PFEGammaAlgo::ProtoEGObject& RO) {
     posZ += cluseraw * cluspos.Z();
     // update EE calibrated super cluster energies
     if( isEE ) {
+      ePS1 = 0;
+      ePS2 = 0;
+      condP1 = condP2 = 1;
       const auto& psclusters = RO.ecal2ps.at(clus.first);
+   
+      for( auto i_ps = psclusters.begin(); i_ps != psclusters.end(); ++i_ps) {
+        const PFClusterRef&  psclus = i_ps->first->clusterRef();
+
+        const std::vector<reco::PFRecHitFraction> recH_Frac = psclus->recHitFractions();
```

Comment:
see comments in  RecoEcal/EgammaClusterAlgos/src/PFECALSuperClusterAlgo.cc ,
the same applies here